app_test.go

```diff
@@ -1451,7 +1452,7 @@ func TestApp_Run_CommandWithSubcommandHasHelpTopic(t *testing.T) {
 		}
 
 		output := buf.String()
-		//t.Logf("output: %q\n", buf.Bytes())
+		//log.Printf("output: %q\n", buf.Bytes())
```

Comment:
I'd prefer to remove these comment if you don't mind.

Reply:
Done